hack/terraform/devstream-iam.tf

```diff
@@ -129,6 +129,62 @@ resource "aws_iam_group_policy" "devstream-eks" {
   })
 }
 
+resource "aws_iam_group_policy" "DevStream-Download-Bucket-RW-Policy" {
+  name  = "DevStream-Download-Bucket-RW-Policy"
+  group = aws_iam_group.devstream.name
+
+  policy = jsonencode({
+    "Version": "2012-10-17",
+    "Statement": [
+      {
+        "Sid": "VisualEditor0",
+        "Effect": "Allow",
+        "Action": [
```

Comment:
I do not think we need this much permission just to upload to/download from a bucket. Please re-test and ensure only the minimum required permission.

Reply:
Limit the minimum available permissions in terraform